examples/logs-slack-alerts/README.md

```diff
@@ -0,0 +1,23 @@
+# Logging Slack Alerts
+
+This logging slack alerts example module schedules a job to run hourly queries of any errors which have occurred in logs which have been ingested into BigQuery. If any errors are found, the errors are sent as alerts to a slack webhook.
+
+Running this module requires log exports into BigQuery in the specified project/region, which is not handled by this example. 
+A good example of exported logging in BigQuery can be found in [Stackdriver Logging](https://cloud.google.com/logging/docs/export/).
+
+[^]: (autogen_docs_start)
+
```

Comment:
Somewhere you should include the IAM roles that are required for this Cloud Function to run

Reply:
Agreed. We should also add a disclaimer that this example does not have test coverage and may be unintentionally broken by future releases to the module.